pkg/cli/build.go

```diff
@@ -68,6 +69,7 @@ bill of materials) describing the image contents.
 				build.WithExtraKeys(extraKeys),
 				build.WithTags(args[1]),
 				build.WithExtraRepos(extraRepos),
+				build.WithArch(types.Architecture(buildArch)),
```

Comment:
Is there any sense in pulling out and reusing some of [`types.ParseArchitectures`](https://github.com/chainguard-dev/apko/blob/9f2503770798549dcb8f05ca319b3d7e087104ef/pkg/build/types/types.go#L122) so callers can use the apk-style arch strings? e.g., `--build-arch=x86_64`

Reply:
Yeah, go ahead if you want to refactor it :)